src/main/groovy/io/openliberty/tools/gradle/tasks/AbstractFeatureTask.groovy

```diff
@@ -297,7 +296,7 @@ public class AbstractFeatureTask extends AbstractServerTask {
     private void createNewInstallFeatureUtil(Set<String> pluginListedEsas, List<ProductProperties> propertiesList, String openLibertyVerion, String containerName, List<String> additionalJsons, Collection<Map<String,String>> keyMap) throws PluginExecutionException {
         try {
 			logger.info("Feature signature verify option: " + server.features.verify)
-            util = new InstallFeatureTaskUtil(getInstallDir(project), project.getBuildDir(), server.features.from, server.features.to, pluginListedEsas, propertiesList, openLibertyVerion, containerName, additionalJsons, server.features.verify, keyMap)
+            util = new InstallFeatureTaskUtil(getInstallDir(project), project.getLayout().getBuildDirectory().getAsFile().get(), server.features.from, server.features.to, pluginListedEsas, propertiesList, openLibertyVerion, containerName, additionalJsons, server.features.verify, keyMap)
```

Comment:
This one makes sense to me. So why do you use `project.layout.buildDirectory.asFile.get()` in some places, and this in others? Is it safer to use the get methods?